app/react/portainer/system/useSystemInfo.ts

```diff
@@ -9,8 +9,8 @@ import { queryKeys } from './query-keys';
 export const queryKey = [...queryKeys.base(), 'info'] as const;
 
 export type ContainerPlatform =
-  | 'Docker Standalone'
   | 'Docker Swarm'
+  | 'Docker Standalone'
   | 'Kubernetes'
   | 'Podman';
 
```

Comment:
Does the order matter? If no, revert the changes. If yes, add a code comment to explain.